src/specBuilder/bullet/bulletSpecBuilder.ts

```diff
@@ -31,9 +31,11 @@ export const addBullet = (
         dimension,
         target,
         color = DEFAULT_COLOR,
+        direction,
```

Comment:
```suggestion
        direction = DEFAULT_BULLET_DIRECTION,
```

Reply:
You will need to add this to the `constants.ts` file for this to work. This will default the value of direction to `vertical` if it isn't defined.